webrtc.html

```diff
@@ -6082,6 +6082,55 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <a><code>RTCRtpTransceiver</code></a>. The sender of this
               transceiver will no longer send, the receiver will no longer
               receive, and the <a>negotiation-needed flag</a> is set.</p>
+              <p>Stopping a transceiver will cause future calls to
+              <code>createOffer</code> to generate a zero port in the
+              <a>media description</a> for the corresponding transceiver,
+              as defined in <span data-jsep="stop">[[!JSEP]]</span>.</p>
+              <p>When this method is invoked, the User Agent MUST run the
+              following steps:</p>
+              <ol>
+                <li>
+                  <p>Let <var>transceiver</var> be the 
+                  <code><a>RTCRtpTransceiver</a></code> object which is to
+                  be stopped.</p>
+                </li>
+                <li>
+                  <p>If <code><var>transceiver</var>.stopped</code> is <code>true</code>
+                  abort these steps.</p> 
+                </li>
+                <li>
+                  <p>Let <var>connection</var> be the
+                  <code><a>RTCPeerConnection</a></code> object on which
+                  the <var>transceiver</var> is to be stopped.</p>
+                </li>
+                <li>
+                  <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+                  <code>true</code>, throw an <code>InvalidStateError</code>
+                  exception and abort these steps.</p>
+                </li>
+                <li>
+                  <p>Let <var>sender</var> be <code><var>transceiver</var>.sender</code>.</p>
+                </li>
+                <li>
+                  <p>Let <var>receiver</var> be <code><var>transceiver</var>.receiver</code>.</p>
+                </li>
+                <li> 
+                  <p><var>sender</var> will no longer send media.</p>
```

Comment:
May be just a matter of preference, but since this is in a list of steps it feels like it should be an instructive rather than declarative statement. So I'd prefer "Stop sending media with sender" (or "permanently stop", to be more definitive).